operators.go

```diff
@@ -11,22 +11,24 @@ import (
 	"github.com/Masterminds/squirrel"
 )
 
-type ScalarCond func(col SchemaField, value interface{}) Condition
+// ScalarCond returns a kallax.Condition that compares a property with the passed
+// values, considering its scalar values (eq, gt, gte, lt, lte, neq)
+type ScalarCond func(col SchemaField, n interface{}) Condition
 
 // Condition represents a condition of filtering in a query.
 type Condition func(Schema) squirrel.Sqlizer
 
 // Eq returns a condition that will be true when `col` is equal to `value`.
-func Eq(col SchemaField, value interface{}) Condition {
+func Eq(col SchemaField, n interface{}) Condition {
```

Comment:
I would not change this to `n` as this implies it's a number, but it's not. `Eq` can be used with almost all types.